sdk/mongocluster/tests.mgmt.yml

```diff
@@ -0,0 +1,16 @@
+trigger: none
+
+pr: none
+
+extends:
+  template: /eng/pipelines/templates/stages/archetype-sdk-tests.yml
+  parameters:
+    ServiceDirectory: mongocluster
+    Artifacts:
+      - name: azure-resourcemanager-mongocluster
+        groupId: com.azure.resourcemanager
+        safeName: azureresourcemanagermongocluster
+    Clouds: 'Public'
+    # Only run tests on Windows to save cost.
```

Comment:
@weidongxu-microsoft Do we need `useFederatedAuth: true`?

Reply:
I think it is "yes".

Reply:
@v-hongli1 Please use the latest template in your previous PR:
https://github.com/Azure/azure-sdk-for-java/pull/40942/files